package.json

```diff
@@ -62,6 +62,11 @@
     "knex": "0.13.0",
     "knex-migrator": "2.0.16",
     "lodash": "4.17.4",
+    "markdown-it": "8.3.1",
+    "markdown-it-footnote": "3.0.1",
+    "markdown-it-lazy-headers": "^0.1.3",
```

Comment:
Oops, I must have missed the `--exact` flag when adding that one. Agree that looser version numbers make sense now that we have `yarn.lock` - `ember-cli` will use the locked versions and we still generate a `shrinkwrap.json` for the release too.

I'll pin for this PR and we can discuss a wider change elsewhere 😄